models/test_config.json

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
there's a new file `models/test_config.json` - I guess this was created by running the tests? The format of the file is different to `models/cell_config.json` and `models/particle_config.json`. It looks like `test_config.json` is a new-style config file, and the other two are the [legacy format](https://github.com/quantumjot/btrack/blob/main/btrack/config.py#L146). Is it worth keeping `test_config.json` so we have an example new-style config? Should it be renamed to something else?

Reply:
If it's created in the tests, should this be ignored?

Reply:
usually yeah, but it's also useful to know what the new-style config format is, as the other examples use the legacy format. So perhaps we could keep this file and rename it to something more descriptive, then add `models/test_config.json` to .gitignore?

Reply:
Sounds good